### PR TITLE
[inventory] use saml who.ini

### PR DESCRIPTION
**Pull request by adborden**
Missed this
https://github.com/GSA/datagov-deploy/pull/2752#discussion_r571242356

### CODE REVIEW

**Review by jbrown-xentity — APPROVED** [review]

File: ansible/inventories/production/group_vars/inventory-next/vars.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ inventory_ckan_plugins_additional: [s3filestore saml2auth]
 # saml2 authentication
 inventory_ckan_saml2_enabled: true
 ckan_production_ini_template: etc/ckan/production.ini
```
Comment (FuhuXia) [inline ansible/inventories/production/group_vars/inventory-next/vars.yml:42]: also this.
ckan_production_ini_template: inventory-next/etc_ckan_production.ini.j2

Let me make a PR.

Comment (FuhuXia) [reply ansible/inventories/production/group_vars/inventory-next/vars.yml:42]: https://github.com/GSA/datagov-deploy/pull/2763

File: ansible/inventories/production/group_vars/inventory-next/vars.yml [hunk 2]
```diff
@@ -40,7 +40,7 @@ inventory_ckan_plugins_additional: [s3filestore saml2auth]
...
-inventory_ckan_who_ini_path: etc_ckan_who.default.ini.j2
+inventory_ckan_who_ini_path: etc_ckan_who.saml2.ini.j2
```
Comment (FuhuXia) [inline ansible/inventories/production/group_vars/inventory-next/vars.yml:43]: `etc_ckan_who.saml2.ini.j2` is who.ini for ckanext-saml2 for max.log.
`inventory-next/etc_ckan_who.saml2.ini.j2` is who.ini for ckanext-saml2auth for login.gov

Let me make another PR.

Comment (FuhuXia) [reply ansible/inventories/production/group_vars/inventory-next/vars.yml:43]: https://github.com/GSA/datagov-deploy/pull/2765
